tests/unit/systems/ops/nvtabular/test_op.py

```diff
@@ -29,7 +29,8 @@
 from nvtabular import ops as wf_ops  # noqa
 
 ensemble = pytest.importorskip("merlin.systems.dag.ensemble")
-workflow_op = pytest.importorskip("merlin.systems.dag.ops.workflow")
+wf_op = pytest.importorskip("merlin.systems.dag.ops.workflow")
```

Comment:
Should this be moved to the triton ops directory, since it tests the triton op behaviors?

Reply:
Yeah, that's probably a good idea 👍🏻